celery/bin/worker.py

```diff
@@ -45,8 +45,20 @@ def __init__(self):
     def convert(self, value, param, ctx):
         # Pools like eventlet/gevent needs to patch libs as early
         # as possible.
-        return concurrency.get_implementation(
-            value) or ctx.obj.app.conf.worker_pool
+        value = super().convert(value, param, ctx)
```

Comment:
and this part of changes?

Reply:
We don't have tests for the CLI as of yet.